rest/route/middleware.go

```diff
@@ -394,40 +394,46 @@ func urlVarsToProjectScopes(r *http.Request) (string, int, error) {
 	}
 
 	projectID := util.CoalesceStrings(append(query["project_id"], query["projectId"]...), vars["project_id"], vars["projectId"])
+	destProjectID := util.CoalesceString(query["dest_project"]...)
+
+	res := []string{projectID}
```

Comment:
ProjectID can be set at any point in the function, so you need to do this at the bottom